### ISSUE TITLE
The prop value with an expression type of TSNonNullExpression could not be resolved.

### ISSUE THREAD

**Issue by Ovyerus** [jsx-eslint/jsx-ast-utils#82]
Found this happening with `<MyComponent role={role!}/>`
Doesn't seem to happen with other props in my (limited) testing, ala #70 

```
Error: The prop value with an expression type of TSNonNullExpression could not be resolved. Please file issue to get this fixed immediately.
Occurred while linting E:\project\src\pages\signup.tsx:189
    at Array.forEach (<anonymous>)
    at Array.forEach (<anonymous>
```

**Comment by jessebeach** [comment]
@Ovyerus , Fixed in version v2.2.1. This will log an error in the console, but it will no longer throw.

**Comment by ljharb** [comment]
We should still keep these open until we are actually handling the nodes.
